openshift-templates/s2i-app-build/template-no-secrets.json

```diff
@@ -54,9 +54,33 @@
                     "type": "JenkinsPipeline"
                 },
                 "triggers": [
-                    {
-                        "type": "ConfigChange"
+                  {
+                    "type": "ConfigChange"
+                  },
+                  {
+                    "type": "GitHub",
+                    "github": {
+                      "secret": "[A-Z0-9]{4}"
                     }
+                  },
+                  {
+                    "type": "Generic",
+                    "generic": {
+                      "secret": "[A-Z0-9]{4}"
+                    }
+                  },
+                  {
+                    "type": "GitLab",
+                    "gitlab": {
+                      "secret": "[A-Z0-9]{4}"
+                    }
+                  },
+                  {
+                    "type": "Bitbucket",
+                    "bitbucket": {
+                      "secret": "[A-Z0-9]{4}"
```

Comment:
needs to updated a la [this example](https://github.com/openshift/rails-ex/blob/master/openshift/templates/rails-postgresql.json#L510)